character.cpp

```diff
@@ -47,3 +47,68 @@ std::ostream & operator<<(std::ostream & os, const Character &C) {
 	os << C.getName() << ": HP: " << C.getHp() << ", DMG: " << C.getDmg() << '\n';
 	return os;
 }
+
+Character Character::parseUnit(std::string charSheetName)
+{
+	std::fstream charSheet(charSheetName);
+
+	if (charSheet.fail())
+	{
+		std::string error("Couldn't open file");
+		throw std::runtime_error(error);
+	}
+
+	std::string name = "";
+	int hp = 0;
+	int dmg = 0;
+
+	std::string line;
+
+	while (!charSheet.eof())
+	{
+		std::getline(charSheet, line);
+
+		if (name == "")
+		{
+			std::string s = "name";
+			if (line.find(s) != std::string::npos) {
```

Comment:
hasznalod meg barhol mashol az s-t? ha nem, akkor nem lett volna egyszerubb egy sorban?

Reply:
Sot, az a ket if is mehet egyetlen sorba, es akkor meg atlathatobb a kod.